python/py-pdb2pqr/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               select 1.0
+PortGroup               github 1.0
+name                    py-pdb2pqr
+version                 3.6.1
+revision                0
+
+supported_archs         noarch
+platforms               {darwin any}
+license                 BSD
+maintainers             {gmail.com:howarth.at.macports @jwhowarth} openmaintainer
+
+description             Preparation of protein structures for electrostatics calculations
+long_description        {*}${description}
+
+homepage                https://github.com/Electrostatics/pdb2pqr
+github.setup            Electrostatics    pdb2pqr    ${version} v
```

Comment:
is there a reason this needs to come from GitHub? The preferred source for Python packages is PyPI and would thus only use the `python` PG. 

If there is a compelling reason to use the `github` PortGroup you should do so correctly. For example, the `version` should not be a separate line but included in the `github.setup` line, entries there should be separated by single space, `homepage` is set automatifcally as well as `master_sites`. You cannot set `github.tarball_from` to `releases` and `archive`; pick one; `distfiles` is set automatically by the PortGroup, etcetera... 

This comment applies to all ports in this PR.

Reply:
I ended up using GitHub to get a copy of pdb2pqr which had the tests files in unix rather than dos format. The PyPi copy has the extra line feeds which fouls up applying skip-network-tests.patch. I assume that if I have to use PyPi, I'll have to add build dependency on dos2unix and run each of the offending tests files through it in pre-patch to strip out those DOS LFs.

Reply:
According to this web page [https://stackoverflow.com/questions/2076688/how-to-use-patches-created-in-windows-with-crlf-in-linuxi](url), there isn't a way to get linux patch to deal with dos files that have CRLF short of using dos2unix on them first. I'd rather not have to add a dos2unix dependency on the pdb2pqr package when a GitHub for the code exists that is in unix rather than dos format.

Reply:
Regarding the use of GitHub for the propka 3.5.1 sources, the test suite of the copy on PyPi is broken whereas the one on GitHub works fine. Unfortunately, pdb2pqr and propka have a lot of local branches with fixes that apparently don't get back ported.

Reply:
Likewise, the mmcif-pdbx on PyPi is missing the test suite that the copy on GitHub has.

Reply:
that's fine. As I said, if there is a reason to use GitHub that's certainly fine. 

Please address the other comments.